app/controllers/homepage_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HomepageController < ApplicationController
+  include HomepageHelper
+  def index
+    @companies = Company.all
+    @categories = Category.all
+    breakdown_jobs_by_level_of_interest
```

Comment:
i like that you've extracted out these methods, but these methods belong in the model rather than the view helper 👍